examples/demo_dlmbl/python/excercise_1.py

```diff
@@ -0,0 +1,274 @@
+
+The layout on the disk is: row/col/field/resolution/timepoint/channel/z/y/x.
+
+
+Note that labelling is not perfect,
+as some cells are not expressing the fluorophore.
+"""
+
+# %%
+
+row = "0"
+col = "0"
+field = "0"
+resolution = "0" # 0 is the highest resolution, 1 is 2x2 binned, 2 is 4x4 binned, etc.
+image = dataset[f'{row}/{col}/{field}/{resolution}'].numpy()
+print(image.shape)
+
+figure, axes = plt.subplots(1, 3, figsize=(9, 3))
+
+for ax, channel in zip(axes, image[0, :, 0]):
+    ax.imshow(channel, cmap="gray")
+    ax.axis("off")
+
+plt.tight_layout()
+
+# %% [markdown]
+"""
+Configure the data loaders for training and validation.
+"""
+
+# %%
+data_module = HCSDataModule(
+    data_path,
+    source_channel="Phase",
+    target_channel=["Nuclei", "Membrane"],
+    z_window_size=1,
+    split_ratio=0.8,
+    batch_size=BATCH_SIZE,
+    num_workers=8,
+    architecture="2D",
+    yx_patch_size=(256, 256),
+)
+
+data_module.setup("fit")
+
+print(len(data_module.train_dataset), len(data_module.val_dataset))
+
+# %% [markdown]
+"""
+<div class="alert alert-info">
+Task 1.2
+
+Validate that the data can be loaded in batches correctly.
+</div>
+"""
+
+# %%
+train_dataloader = data_module.train_dataloader()
+
+for i, batch in enumerate(train_dataloader):
+    ...
+    # check some batches and break
+    break
+
+# %% tags=["solution"]
+train_dataloader = data_module.train_dataloader()
+
+
+fig, axs = plt.subplots(3, 8, figsize=(20, 6))
+
+# Draw 8 batches, each with 32 images. Show the first image in each batch.
+
+for i, batch in enumerate(train_dataloader):
+    # The batch is a dictionary consisting of three keys: 'index', 'source', 'target'.
+    if i >= 8:
+        break
+    FOV = batch['index'][0][0]
+    input_tensor = batch['source'][0, 0, :, :].squeeze()
+    target_membrane_tensor = batch['target'][0, 0, :, :].squeeze()
+    target_nuclei_tensor = batch['target'][0, 1, :, :].squeeze()
+
+    axs[0, i].imshow(input_tensor, cmap='gray')
+    axs[1, i].imshow(target_nuclei_tensor, cmap='gray')
+    axs[2, i].imshow(target_membrane_tensor, cmap='gray')
+    axs[0, i].set_title(f'input@{FOV}')
+    axs[1, i].set_title('target-nuclei')
+    axs[2, i].set_title('target-membrane')
+    axs[0, i].axis('off')
+    axs[1, i].axis('off')
+    axs[2, i].axis('off')
+
+plt.tight_layout()
+plt.show()
+
+
+# %% [markdown]
+"""
+Construct a 2D U-Net for image translation.
+
+See ``viscy.unet.networks.Unet2D.Unet2d`` for configuration details.
+Increase the ``depth`` in ``draw_graph`` to zoom in.
+"""
+
+# %%
+model_config = {
+    "architecture": "2D",
+    "residual": True,
+    "dropout": 0.1,
+    "task": "reg",
+}
+
+model = VSUNet(
+    model_config=model_config.copy(),
+    batch_size=BATCH_SIZE,
+    loss_function=torch.nn.functional.mse_loss,
+    schedule="WarmupCosine",
+    log_num_samples=10,
+)
+
+# visualize graph
+model_graph = draw_graph(model, model.example_input_array, depth=2)
+graph = model_graph.visual_graph
```

Comment:
love this graph!

Reply:
The graph's output tensor is 1x1x256x256. Is this the artifact of shape of `example_input_tensor`. Which paramter specifies the number of output channels in model arch?

Reply:
Sorry this is a bug. Now fixed.